packages/contracts-bedrock/contracts/test/MerkleTrie.t.sol

```diff
@@ -358,10 +302,16 @@ contract MerkleTrie_Test is CommonTest {
         proof[3] = hex"c32081aa";
 
         vm.expectRevert("MerkleTrie: value node must be last node in proof (leaf)");
-        MerkleTrie.get(
-            key,
-            proof,
-            root
-        );
+        MerkleTrie.get(key, proof, root);
+    }
+
+    /// @notice The `bytes4` parameter is to enable fuzz runs; it is ignored.
```

Comment:
Adding a negative case would be great too, to fuzz invalid proofs passing because that would be really bad

Reply:
Will do today :smile: